neptune/internal/notebooks/notebooks.py

```diff
@@ -29,17 +29,20 @@ def create_checkpoint(backend, notebook_id, notebook_path):
         # pylint:disable=bad-option-value,import-outside-toplevel
         import IPython
         ipython = IPython.core.getipython.get_ipython()
-        execution_count = ipython.kernel.execution_count
+        execution_count = -1
+        if ipython is not None and ipython.kernel is not None:
+            execution_count = ipython.kernel.execution_count
         with _checkpoints_lock:
 
-            if execution_count in _checkpoints:
+            if execution_count in _checkpoints and execution_count != -1:
```

Comment:
Is second condition necessary? `execution_count in _checkpoints` implies `execution_count != -1`, right?

Reply:
Agreed, changed